compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -269,6 +270,14 @@ impl<'tcx> CodegenUnit<'tcx> {
         self.name = name;
     }
 
+    pub fn primary(&self) -> bool {
```

Comment:
nit: `is_primary`? `primary` sounds like a weird name to have for an accessor  of a field called `is_primary`.

Reply:
Sure! I swapped the two.